src/blog/crud.py

```diff
@@ -0,0 +1,65 @@
+import dataclasses
+import logging
+from datetime import date, datetime
+from typing import Optional
+
+import database
+import sqlalchemy
+
+# from auth.models import SiteUser
+from blog.models import BlogPosts
+from sqlalchemy import func
+
+_logger = logging.getLogger(__name__)
+
+async def create_new_entry(db_session: database.DBSession,
+                            title:str, computing_id:str, post_tags: str, html_content:str,
+                            date_created:date, last_edited: date):
+
+    """ To create a new blog entry """
+
+    return None
+
+async def fetch_by_title(db_session: database.DBSession, title:str) -> (str, str, datetime, list[str] | None) | None:

```

Comment:
```suggestion
async def fetch_by_title(db_session: database.DBSession, title:str) -> tuple[str, str, datetime, list[str] | None] | None:
```
unfortunately tuple type hints are not formatted the obvious way (I hope they switch it one day, but I doubt it)